frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-react": "^7.34.1",
-    "eslint-plugin-storybook": "^0.8.0",
+    "eslint-plugin-react": "^7.37.3",
+    "eslint-plugin-storybook": "^0.11.2",
+    "globals": "^15.14.0",
```

Comment:
needed an upgrade for envs newer than es2015, but the config is now set to all builtins/latest anyway and it *seems* fine.